### ISSUE TITLE
Getting error 'QuestionAnsweringModelOutput has no attribute logits' in Span Extraction.ipynb

### ISSUE THREAD

**Issue by ShreyasDongre-02** [veselink1/sponsorml-ytcaptions#1]
Hey,
I tried to execute the code for span extraction and the evaluate() for entire video transcript works as intended. But later down in the final evaluate() which calls predict_in_batches() there I am getting error at line: `predictions = torch.nn.functional.softmax(outputs.logits, dim=-1).tolist()` 

I was also getting error at `outputs = trained(**inputs)` saying "Got unexpected input Offset mapping". I resolved it by passing only input ids to the model. Thats when i got this current error.
